Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -2111,6 +2267,11 @@ def get_modified_remote_data_command(args: Dict[str, Any]):
     for detection_id in raw_detections:
         modified_ids_to_mirror.append(str(detection_id))
 
+    raw_idp_detections = get_idp_detections_ids(filter_arg=f"updated_timestamp:>'{last_update_timestamp}'+product:'idp'",
```

Comment:
last_update_timestamp is in milliseconds or seconds? Do we need to worry about the issue of multiple incidents at the same time?

Reply:
In milliseconds, also.. we're using lookback and I'm considering adding offset calculations